docs/schema/guidance/record-ids.rst

```diff
@@ -4,4 +4,53 @@
 Record identifiers
 ==================
 
-[Stub page]
+To publish beneficial ownership data in BODS format, an information management system MUST maintain record identifiers for each element (person, entity and relationship) in each beneficial ownership network.
+
+Record identifiers MUST be unique within the publisher's system and never reused to refer to a different element. 
+
+There are situations where a publisher's system can allow information about the same person, entity or relationship to be collected in relation to the beneficial ownership networks of two (or more) different subjects. In these cases, information from different sources about the same person, entity or relationship can exist side-by-side and needs to be represented in published data. In these cases, each set of information MUST have its own record identifier. (See the example below.)
```

Comment:
I'm finding this a bit difficult to understand - I think because it's talking both about overlapping networks and different sources. Is the multiple sources the key aspect here? If so maybe reword along these lines?

'Information about one element MAY be submitted by multiple sources. Each set of information about an element from a different source MUST have its own record identifier. (See the example below.)'

Reply:
> Is the multiple sources the key aspect here?

It's really the overlapping networks (in the sense of them containing the same element) that is the key. I'm going to remove the reference to 'sources' since that's a bit of a red herring.